src/txmempool.h

```diff
@@ -118,6 +118,10 @@ class CTxMemPool
     void removeConflicts(const CTransaction &tx, std::list<CTransaction>& removed);
     void removeForBlock(const std::vector<CTransaction>& vtx, unsigned int nBlockHeight,
                         std::list<CTransaction>& conflicts, bool fCurrentEstimate = true);
+    // evict a random transaction and its descendants, given a random transaction ID,
+    // BUT does not evict prioritised transactions. Returns list of transactions
+    // evicted.
+    void evictRandom(const uint256& hash, std::list<CTransaction>& evicted);
```

Comment:
Why is the "random transaction ID" here passed as an argument? Wouldn't it make more sense for evictRandom to generate the random key itself?